### ISSUE TITLE
[gap-analysis] images on github issues are in githubusercontent even for /TR/

### ISSUE THREAD

**Issue by himorin** [w3c/i18n-activity#1023]
Like [images in Japanese gap-analysis at /TR/](https://www.w3.org/TR/jpan-gap/#issue180_vertical_text), images inserted to source github issues will be compiled to refer ones in githubusercontent with current script for respec.
It's not an easy issue to resolve by the script, that even we can get a list of images like that (surely can from a script for respec), we need to have a way to upload these to /TR/.

**Comment by xfq** [comment]
Indeed, it would be better if these images can be on w3.org (and/or at least in the GitHub repo), because we don’t know how long they can be kept in githubusercontent.

(The same is true for the images in [type-samples](https://github.com/w3c/type-samples), although they're not published on /TR.)
